### DISCUSSION TITLE
EA

### DISCUSSION THREAD

**Discussion by edzob** [JRBliekendaal/master-thesis#58] [Ideas]
From a paper I am working on:
"As stated by many, architecturing needs to deliverperformance and value by facilitating good decisionmaking (van den Berg et al.,2019). " 

van den Berg, M., Slot, R., van Steenbergen, M., Faasse,P., and van Vliet, H. (2019). How enterprise architec-ture improves the quality of IT investment decisions.152:134–150.https://linkinghub.elsevier.com/retrieve/pii/S0164121219300433.

Artikel is nav zijn Promotie dissertatie/thesis
https://research.vu.nl/ws/portalfiles/portal/85088972/complete+dissertation.pdf

**Comment by JRBliekendaal** [comment]
> From a paper I am working on:
> 
> "As stated by many, architecturing needs to deliverperformance and value by facilitating good decisionmaking (van den Berg et al.,2019). " 
> 
> 
> 
> van den Berg, M., Slot, R., van Steenbergen, M., Faasse,P., and van Vliet, H. (2019). How enterprise architec-ture improves the quality of IT investment decisions.152:134–150.https://linkinghub.elsevier.com/retrieve/pii/S0164121219300433.
> 
> 
> 
> Artikel is nav zijn Promotie dissertatie/thesis
> 
> https://research.vu.nl/ws/portalfiles/portal/85088972/complete+dissertation.pdf
> 
> 

Will read it. Not this weekend. Quality time with family. 😬

**Comment by edzob** [comment]
zeker niet helemaal gaan lezen . voor ea definities  zijn er duizenden
alternatieven 😊

Edzo

-- 
"It is amazing what you can accomplish if you do not care who gets the
credit." - Harry S Truman

Mobile, Signal, WhatsApp -
+31629485425
Twitter, riot.im, wire, Matrix -
@edzob

Homepage - https://www.edzob.com/page/about
ORCID - https://orcid.org/0000-0003-0097-7375

**Reply by JRBliekendaal** [reply]
because EA is one of my main concepts I have to be more carefully when I select a definition. So selecting the right mental model is in this case important.

Approach is do a little small research for the background to define it and search for the one that has a quality stamp (peer reviewed etc...) that could help in my context.

**Comment by edzob** [comment]
Sounds smart and with AMS the right place.
For the school of AMS, The book of Op t land is pretty good,
Hoogervorst lankhorst and many more wrote stuff about this.

recent Research are from Nurmi, lapalme and Kotusev 
Where Nurmi his dissertation seems relevant and very new :)

Nurmi, J. (2021). Enterprise architecture in public sector
ecosystems: A systems perspective. PhD thesis, Uni-
versity of Jyväskylä.
http://urn.fi/URN:ISBN:978-951-39-8518-9.
